derived-tables.sql

```diff
@@ -10,7 +10,8 @@ create table duplicate_record_ids as
             record_barcodes
         group by
             project_id,
-            record_id
+            record_id,
+            repeat_instance
```

Comment:
Repeat instance is only unique within an event, so the grouping key needs to be (project_id, record_id, event_name, repeat_instance).

Reply:
In adding this, I realized we were incorrectly storing the event name in the "record_arm" key in the produced ndjson file of barcodes. I fixed this, too, in a new commit.